src/utils/isOnlyEmoji.ts

```diff
@@ -1,8 +1,35 @@
 const emojiRegex =
-  /(<a?(:\w+:\d+)>|\p{Emoji_Presentation}|\p{Extended_Pictographic}|\p{Emoji_Component})\s*/gu
+  /(<a?(:\w+:\d+)>|\p{Emoji_Presentation}|\p{Extended_Pictographic}|\p{Emoji_Component}|:\w+:\s*)/gu
 
 export default (msg: string): boolean => {
   const emoji = msg.match(emojiRegex)
+  //the issue of #99(https://github.com/creatorsgarten/kaogeek-discord-bot/pull/99) only happend with only number message,
+  //so detect number in message and return false if there is.
+  if (emoji !== null) {
+    const unicoded = emoji.map((emo) => {
+      return emo.codePointAt(0)?.toString(16)
+    })
+    for (const i of unicoded) {
+      if (i !== undefined && isNumber(i)) {
```

Comment:
from this https://github.com/kaogeek/kaogeek-discord-bot/pull/109#discussion_r1202093527 
I am sorry, I missed that case. btw, I think we can add `i.length === 2` to resolve that problem
```suggestion
      if (i !== undefined && i.length === 2 && /^\d+$/.test(i)) {
```
This way will improve performance significantly,
but its fine if you want to use your version. I just suggest it.